libcaf_core/caf/string_algorithms.cpp

```diff
@@ -54,9 +54,9 @@ void split(std::vector<std::string_view>& result, std::string_view str,
 std::string_view trim(std::string_view str) {
   auto non_whitespace = [](char c) { return !isspace(c); };
   if (std::any_of(str.begin(), str.end(), non_whitespace)) {
-    while (str.front() == ' ')
+    while (std::isspace(str.front()))
```

Comment:
Nit: For all the functions from `<cctype>`, you technically need to cast to `unsigned char` [to avoid UB](https://en.cppreference.com/w/cpp/string/byte/isspace).

Reply:
You are of course correct, thanks!